### ISSUE TITLE
[Enhancement] Avenger speed on geoscape

### ISSUE THREAD

**Issue by ansabalirai** [long-war-2/lwotc#530]
A low priority enhancement, but with the amount of flying needed in LWOTC between missions, it might be good to increase the movement speed of the avenger. By that, I do not just mean animations speed but how time elapsed in-game between flights. While LW2 had this in place, it seems to be missing in LWOTC.

**Comment by pledbrook** [comment]
Do you have an example of what you mean? I don't really know what I should be looking for.

**Comment by allenhu0912** [comment]
Say you are scanning in East Asia when a mission pops up in Mexico. You want to send a team to infiltrate, so the Avenger flies from East Asia to Mexico. The trip from East Asia to Mexico does not happen instantly. The in-game time will advance by a couple of hours (but remember to account for the time zone difference). This means that if you have different squads infiltrating all across the world, and you need to recall all of them at once to go on that invasion mission with 3 hours left on the timer, you will probably not have enough time because the Avenger will spend way more than 3 hours flying around over the world

**Comment by pledbrook** [comment]
That was how LW2 behaved, I'm pretty sure. Are you saying LW2 eliminated the travel time?

**Comment by allenhu0912** [comment]
The OP seems to believe that the Avenger was faster in LW2. Not instantaneous travel, but faster. I don't know, I haven't measured it.

**Comment by ansabalirai** [comment]
Yes, my thinking was that the faster movement in LW2 also meant that the time elapsed was less than an equivalent trip in WOTC vanilla. It is hard to verify since the timer kinda goes weird when the avenger initiates movement. But even if the time taken is the same in both cases, the flying animation speed is definitely much faster in vanilla LW2

**Comment by pledbrook** [comment]
Are you sure LWOTC is slower than original LW2? tracktwo pointed out the config settings, and they are certainly in the LWOTC INIs. I can't see any other patches in the old LW2 code.

**Comment by pledbrook** [comment]
And so I remember when I come back to this issue, look for `Acceleration=` in the _XComGameData.ini_ and _XComGameBoard.ini_ files.
